src/audio-player-composable/__tests__/audio-player-composable.test.tsx

```diff
@@ -241,6 +257,36 @@ describe('Audio Player Composable', () => {
     expect(audioElement.paused).toBe(true);
   });
 
+  it('should have play or stop label when live', () => {
+    const {getByTestId} = renderWithTheme(
+      AudioPlayerComposable,
+      liveAudioProps,
+    );
+
+    const playPauseButton = getByTestId('audio-player-play-pause-button');
+
+    fireEvent.canPlay(getByTestId('audio-element'));
+    fireEvent.click(playPauseButton);
+    expect(playPauseButton.getAttribute('aria-label')).toBe('Stop');
+    fireEvent.click(playPauseButton);
+    expect(playPauseButton.getAttribute('aria-label')).toBe('Play');
+  });
+
+  it('should have play or pause label when live', () => {
```

Comment:
I think this is a mistake?
```suggestion
  it('should have play or pause label when recorded', () => {
```